.github/workflows/needs-more-info.yaml

```diff
@@ -0,0 +1,18 @@
+name: Issue Info Complete
+
+# This workflow is triggered on issue comments.
+on:
+  issue_comment:
+    types: created
+
+jobs:
+  applyNeedsAttentionLabel:
+    name: Apply Info Complete Label
+    runs-on: ubuntu-latest
+    steps:
+      - name: Apply Needs Attention Label
+        uses: hramos/needs-attention@v1
+        with:
+            repo-token: ${{ secrets.GITHUB_TOKEN }}
+            response-required-label: 'need-more-info'
+            needs-attention-label: 'info-completed'
```

Comment:
LGTM, did you consider just adding needs/triage back?

Reply:
I did not but the needs/triage might be misleading since an issue could already be triaged but we still need more info? :thinking: